test/e2e/flowaggregator_test.go

```diff
@@ -1404,6 +1491,26 @@ func matchSrcAndDstAddress(srcIP string, dstIP string, isDstService bool, isIPv6
 	return srcField, dstField
 }
 
+func createToExternalTestPods(t *testing.T, data *TestData) (*PodIPs, string, *PodIPs, string, *PodIPs, string, *PodIPs, func()) {
+	_, serverIPs, serverCleanupFunc := createAndWaitForPod(t, data, func(name string, ns string, nodeName string, hostNetwork bool) error {
+		return data.createServerPod(name, data.testNamespace, "", serverPodPort, false, true)
```

Comment:
just to confirm my understanding: when the destination is a hostNetwork Pod (Pod's IP address is the Node's external IP address), Egress will be applied to the traffic?

probably worth mentioning in a comment

Reply:
Yes, that's what I verified during tests.